Description: make the test less dependent on the pygments version
 Some tests are too strict, and fail when pygments is updated and a
 detail changes in the formatting.
Bug-Debian: https://bugs.debian.org/768615
Bug-Debian: https://bugs.debian.org/775631
Bug-Debian: https://bugs.debian.org/972481
Forwarded: https://github.com/tmm1/pygments.rb/issues/195
Author: Antonio Terceiro <terceiro@debian.org>
Author: Nicolas Boulenguez <nicolas@debian.org>
Author: Per Andersson <avtobiff@gmail.com>
Author: Tomasz Buchert <tomasz.buchert@inria.fr>

--- a/test/test_pygments.rb
+++ b/test/test_pygments.rb
@@ -35,7 +35,8 @@
   def test_highlight_works_with_larger_files
     code = P.highlight(REDIS_CODE)
     assert_match 'used_memory_peak_human', code
-    assert_equal 458511, code.bytesize.to_i
+    # An equality would fail every time pygments change details in formatting.
+    assert 450_000 < code.bytesize.to_i
   end
 
   def test_highlight_works_with_null_bytes
@@ -86,7 +87,8 @@
 
   def test_highlight_works_with_single_character_input
     code = P.highlight("a")
-    assert_match "a\n</pre>", code
+    # With an ambiguous input, the output is unreproducible.
+    # assert_match "a\n</pre>", code
   end
 
   def test_highlight_works_with_trailing_newline
